nativelib/src/main/scala/scala/scalanative/native/package.scala

```diff
@@ -118,4 +119,34 @@ package object native {
     @inline def toUInt: UInt     = toULong.toUInt
     @inline def toULong: ULong   = new ULong(value)
   }
+
+  /** Convert a CString to a java.lang.String */
+  def fromCString(cstr: CString)(implicit charset: java.nio.charset.Charset): String = {
+    val len = string.strlen (cstr).toInt
```

Comment:
Can I have at least a warning in such cases? Currently it looks like everything is fine without any indication of a failure whatsoever even if the file doesn't parse and might be ill-formated.